pkg/kubelet/cm/types.go

```diff
@@ -47,6 +47,17 @@ type CgroupConfig struct {
 	ResourceParameters *ResourceConfig
 }
 
+type MemoryStats struct {
+	// Memory usage (in bytes).
+	Usage int64
+}
+
+// ResourceConfig holds information about all the supported cgroup resource parameters.
+type ResourceStats struct {
```

Comment:
I'd prefer not adding more resource usage types. Let's re-use stats defined under `pkg/kubelet/api/...`